index.js

```diff
@@ -13,9 +14,7 @@ var slotsForTesting = {
   'ORIGIN_ONE': 'DESTINATION_ONE'
 };
 
-// TODO: make dynamic based on environment
-var baseUrl = 'http://54.183.205.82/alexa/';
-// var baseUrl = 'http://localhost:8080/alexa/';
+var baseUrl = config.PROD ? 'http://54.183.205.82/alexa/' : 'http://localhost:8080/alexa/';
```

Comment:
AWS Lambda doesnt support env variables, so this config property would come from a gitignored config.js file in the alexa repo. that config file would be included in the zip that gets uploaded to aws lambda